java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -630,6 +634,34 @@ public CompletableFuture<String[]> lpopCount(@NonNull String key, long count) {
                 response -> castArray(handleArrayResponse(response), String.class));
     }
 
+    @Override
+    public CompletableFuture<Long> lpos(@NonNull String key, @NonNull String element) {
+        return commandManager.submitNewCommand(
+            LPos, new String[] {key, element}, this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<Long> lpos(@NonNull String key, @NonNull String element, @NonNull LPosOptions options) {
+        return commandManager.submitNewCommand(
+            LPos, new String[] {key, element, Arrays.toString(options.toArgs())}, this::handleLongResponse);
```

Comment:
Use `concatenateArrays` in ArrayTransformUtils to merge multiple arrays together.  It's better than using Arrays.toString here.